src/main/java/com/gdschongik/gdsc/domain/coupon/dao/IssuedCouponCustomRepository.java

```diff
@@ -1,11 +1,18 @@
 package com.gdschongik.gdsc.domain.coupon.dao;
 
+import com.gdschongik.gdsc.domain.coupon.domain.CouponType;
 import com.gdschongik.gdsc.domain.coupon.domain.IssuedCoupon;
 import com.gdschongik.gdsc.domain.coupon.dto.request.IssuedCouponQueryOption;
+import com.gdschongik.gdsc.domain.member.domain.Member;
+import com.gdschongik.gdsc.domain.study.domain.Study;
+import java.util.Optional;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
 public interface IssuedCouponCustomRepository {
 
     Page<IssuedCoupon> findAllIssuedCoupons(IssuedCouponQueryOption queryOption, Pageable pageable);
+
+    Optional<IssuedCoupon> findNonRevokedIssuedCouponByCouponTypeAndMemberAndStudy(
```

Comment:
```suggestion
    Optional<IssuedCoupon> findUnrevokedIssuedStudyCoupon(
```

spring data jpa 사용하는게 아니라서, `byCouponTypeAnd...` 는 안붙여줘도 파라미터 단에서 충분히 파악 가능한 컨텍스트일 것 같네요.

Reply:
안 그래도 이 부분 고민하긴 했는데 다른 querydsl 쿼리들을 저런식으로 네이밍한 것 같아서 일단 이렇게 해뒀습니다.
일단 이 부분은 제안 내용으로 수정하고 이외 쿼리들은 여유있을때 별도 이슈에서 처리하겠습니다~

Reply:
굿입니다 이슈 추가부탁드려요